angular/core/services/ahoy_service.coffee

```diff
@@ -1,27 +1,23 @@
-angular.module('loomioApp').factory 'AhoyService', ($rootScope, $location, $timeout) ->
-  if ahoy?
-
-    ahoy.trackClicks();
-    ahoy.trackSubmits();
-    ahoy.trackChanges();
+angular.module('loomioApp').factory 'AhoyService', ($rootScope, $window) ->
+  new class AhoyService
+    init: ->
+      return unless ahoy?
 
-    # ahoy.trackViews does not work in angular, so we fake it
-    $rootScope.$watch ->
-      $location.path()
-    , (path) ->
-      properties =
-        url: $location.absUrl()
-        title: document.title
-        page: $location.path()
+      ahoy.trackClicks()
+      ahoy.trackSubmits()
+      ahoy.trackChanges()
 
-      ahoy.track properties
```

Comment:
This was the primary bug here, we wanted to be doing `ahoy.track '$view', properties`